src/jit/gentree.cpp

```diff
@@ -16852,15 +16865,15 @@ void GenTree::ParseArrayAddress(
             {
                 ValueNum vnForElemSize = vnStore->VNForPtrSizeIntCon(elemSize);
                 ValueNum vnForScaledInx =
-                    vnStore->VNForFunc(TYP_I_IMPL, GetVNFuncForOper(GT_DIV, false), inxVN, vnForElemSize);
+                    vnStore->VNForFunc(TYP_I_IMPL, GetVNFuncForOper(GT_DIV, false, false), inxVN, vnForElemSize);
                 *pInxVN = vnForScaledInx;
             }
 
             if (constInd != 0)
             {
                 ValueNum vnForConstInd = comp->GetValueNumStore()->VNForPtrSizeIntCon(constInd);
-                *pInxVN                = comp->GetValueNumStore()->VNForFunc(TYP_I_IMPL,
-                                                              GetVNFuncForOper(GT_ADD, (gtFlags & GTF_UNSIGNED) != 0),
+                bool     isUnsigned    = ((gtFlags & GTF_UNSIGNED) != 0);
```

Comment:
Use `IsUnsigned()`?

Reply:
There are 31 occurances of (gtFlags & GTF_UNSIGNED) in gentree.cpp alone.
While I agree we should switch over to using IsUnsigned() I will do that for
all occurances in an upcoming  PR.